keyboards/idobo/keymaps/greenshadowmaker/keymap.c

```diff
@@ -0,0 +1,162 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 16
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+};
+
+#define XXXXXXX KC_NO
+
+// Defines the keycodes used by our macros in process_record_user
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/* QWERTY
+ * .--------------------------------------------------------------------------------------------------------------------------------------.
+ * | ESC    | 1      | 2      | 3      | 4      | 5      | -      |        | =      | 6      | 7      | 8      | 9      | 0      | BACKSP |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------|
+ * | TAB    | Q      | W      | E      | R      | T      |        |        | [      | Y      | U      | I      | O      | P      | ]      |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------+--------|
+ * | RAISE  | A      | S      | D      | F      | G      |        | UP     |        | H      | J      | K      | L      | ;      | '      |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------------------------+--------|
+ * | LSHIFT | Z      | X      | C      | V      | B      | LEFT   | DOWN   | RIGHT  | N      | M      | ,      | .      | /      | RSHIFT |
+ * |--------+--------+--------+--------+--------+-----------------+--------+--------+--------+--------+-----------------+--------+--------|
+ * | `      | \      | LALT   | LCTRL  | LOWER  | SPACE  | LGUI   | DEL    | ENTER  | SPACE  | RAISE  |        |        |        | PrtScr |
+ * '--------------------------------------------------------------------------------------------------------------------------------------'
+ */
+
+ [_QWERTY] = { /* QWERTY */
```

Comment:
Could you reformat this to match this: 
https://github.com/qmk/qmk_firmware/blob/master/keyboards/idobo/keymaps/default/keymap.c#L25-L31

Reply:
updated, thanks!!  I missed that the style was different from xd75/default and idobo/default (confusion around board name from massdrop) and had copy/pasted

Reply:
No worries. It's been a fun time getting people sorted on that issue!